static/js/click_handlers.js

```diff
@@ -596,7 +596,7 @@ $(function () {
             name: "settings-toggle",
             values: [
                 { label: "Settings", key: "settings" },
-                { label: "Administration", key: "administration" },
+                { label: "Organization", key: "administration" },
```

Comment:
@timabbott Yeah! That's exactly what I had in my mind, too many renames coming ahead.